ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86-loaded-upgrade-from-4.15.yaml

```diff
@@ -42,16 +43,115 @@ tests:
       CHURN: "false"
       ENABLE_FORCE: "true"
       GC: "false"
+      ITERATION_MULTIPLIER_ENV: "6"
       MAX_UNAVAILABLE: "3"
       WORKER_REPLICA_COUNT: "24"
+      ZONES_COUNT: "3"
     post:
+    - ref: openshift-qe-kube-burner-index
     - chain: ipi-aws-post
     pre:
     - chain: ipi-aws-pre-stableinitial
     - chain: workers-scale
     - chain: openshift-qe-cluster-density-v2
     workflow: openshift-qe-upgrade
   timeout: 8h0m0s
+- always_run: false
+  as: loaded-upgrade-415to416-120nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:initial
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      CHURN: "false"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m5.8xlarge
+      ENABLE_FORCE: "true"
+      GC: "false"
+      ITERATION_MULTIPLIER_ENV: "9"
+      MAX_UNAVAILABLE: "15"
+      PROFILE_TYPE: reporting
+      USER_TAGS: |
+        TicketId 382
+      WORKER_REPLICA_COUNT: "120"
+      ZONES_COUNT: "3"
+    post:
+    - ref: openshift-qe-kube-burner-index
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre-stableinitial
+    - chain: workers-scale
+    - chain: create-infra-move-ingress-monitoring-registry
+    - ref: openshift-qe-cluster-density-v2
+    workflow: openshift-qe-upgrade
+  timeout: 8h0m0s
+- always_run: false
+  as: loaded-upgrade-415to416-180nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:initial
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      CHURN: "false"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m5.8xlarge
+      ENABLE_FORCE: "true"
+      GC: "false"
+      ITERATION_MULTIPLIER_ENV: "9"
+      MAX_UNAVAILABLE: "22"
+      PROFILE_TYPE: reporting
+      USER_TAGS: |
+        TicketId 382
+      WORKER_REPLICA_COUNT: "180"
+      ZONES_COUNT: "3"
+    post:
+    - ref: openshift-qe-kube-burner-index
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre-stableinitial
+    - chain: workers-scale
+    - chain: create-infra-move-ingress-monitoring-registry
+    - ref: openshift-qe-cluster-density-v2
+    workflow: openshift-qe-upgrade
+  timeout: 8h0m0s
+- always_run: false
+  as: loaded-upgrade-415to416-252nodes
```

Comment:
@afcollins @vishnuchalla For loaded upgrades, we are having success at large-scale upgrade tests.
Would like to retain run monthly once